index.bs

```diff
@@ -2887,24 +2944,50 @@ Appeal by Advisory Committee Representatives</h3>
 
 	An [=Advisory Committee representative=] initiates an [=appeal=] by sending a request to the [=Team=],
 	and should also share this request with the [=Advisory Committee=].
-	The request should say “I appeal this Decision”
+	The request <em class=rfc2119>should</em> say “I appeal this Decision”
 	and identify the decision,
-	and may also include their rationale for appealing the decision.
+	and <em class=rfc2119>may</em> also include their rationale for the [=appeal=].
 
 	Note: See [[DECISION-APPEAL inline]] for a recommendation
 	on how to communicate an appeal request to the [=Team=] and the [=AC=].
 
-	Within one week the [=Team=] <em class="rfc2119">must</em> announce the appeal process
+	Within one week, the [=Team=] <em class="rfc2119">must</em> initiate the [=appeal=]
+	in the form of an [=Advisory Committee Override=]
+	on the proposal to overturn the decision.
+
+	The conclusion of the [=Advisory Committee Override=] is final;
+	the same decision cannot be appealed more than once.
+
+	If the [=Advisory Committee Override=]
+	approves the proposal to overturn the decision,
+	those who had initiated the proposal <em class=rfc2119>may</em> revise it
+	to address the causes of rejection
+	and follow the ordinary applicable process
+	to submit the revised proposal.
+
+<h3 id="ac-override">
+Advisory Committee Override</h3>
+
+	An <dfn export>Advisory Committee Override</dfn> is an exceptional two-step procedure
+	used to resolve certain matters where neither the usual [=consensus=]
+	process nor its [[#addressing-fo|escalation path]] is sufficient.
+
+	Note: Currently, this only applies to [=AC Appeals=]
+	and [=votes of no confidence=] in the [=AB=] or [=TAG=].
+
+	First, the [=Team=] <em class="rfc2119">must</em> announce the proposal for an [=Advisory Committee Override=]
 	to the [=Advisory Committee=]
 	and provide a mechanism for [=Advisory Committee representatives=]
-	to respond with a statement of positive support for this appeal.
+	to respond with a statement of positive support for holding a vote.
 	The archive of these statements <em class="rfc2119">must</em> be [=member-only=].
+
 	If, within <span class="time-interval">one week</span> of the Team's announcement,
-	5% or more of the [=Advisory Committee=] support the appeal request,
-	the Team <em class="rfc2119">must</em> organize an appeal vote
+	5% or more of the [=Advisory Committee=] support holding the vote,
```

Comment:
This is basically like having a vote to have a vote.  I don't see the point.

It reduces some of my concerns about DoS, engaging this mechanism is exactly the sort of thing I'd like to avoid, but it also makes it nearly infeasible to use this process.  Even though 5% seems small, that's a pretty significant movement given the activity levels in a pretty large consortium.  I suggested one objector and two seconds, but you could make the two into five or something like that.

Reply:
This is the pre-existing mechanism and threshold used for AC appeals. This attempts to reuse this system, not to change this.

And you're right, I don't think a 5% threshold is small. It is actually a non trivial barrier to meet. But I don't think it should be trivial to launch votes of no confidence (or AC appeals), so I'm personally comfortable with that.

If you do want to insist on a lower threshold, could you clarify if you want that change both for AC appeals AND votes of no confidence (in which case I'd suggest opening a separate issue), or only for votes of no confidence (in which case this is indeed the right place to discuss it)?

Reply:
I'm more concerned about the time limit.  Not only do you have to muster 5% of the membership, you have to do so within a week.

I was viewing this with a lens of the vote of no confidence, but I can't think of a case where this wouldn't be a problem for other such appeals.  I don't want this sort of thing to be easy to trigger, but right now it seems to be akin to impossible.  A healthy system might have this appeal run once every few years, but I can't remember a single occurrence, partly because this requires the moving of mountains.  An accountability system that does not operate is probably one that cannot function, so it might as well not exist.

Reply:
@martinthomson Can you file a separate issue about increasing the time limit? (And fwiw, an AC appeal has occurred at least once, since I recall there was one during the last decade or so.)

Reply:
Filed at https://github.com/w3c/process/issues/1034